File: c#/program14.cs
using System;

struct ArrayX
{
    internal int[] Arr;

    internal ArrayX(int size)
    {
        this.Arr = new int[size];
    }

    public void Accept()
    {
        Console.WriteLine("Enter the elements of the array : ");

        for(int iCnt = 0; iCnt < Arr.Length; iCnt++)
        {
            Arr[iCnt] = int.Parse(Console.ReadLine());
        }
    }

    public void Display()
    {
        Console.WriteLine("Elements of the array are : ");

        for(int iCnt = 0; iCnt < Arr.Length; iCnt++)
        {
            Console.Write(Arr[iCnt]+"\t");
        }
        Console.WriteLine();
    }

    internal bool SearchNum(int search)
    {
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.Length; iCnt++)
        {
            if(Arr[iCnt] == search)
            {
                break;
            }
        }
        if(iCnt == Arr.Length)
        {
            return false;
        }
        else
        {
            return true;
        }
    }
}

struct program14
{
    public static void Main()
    {
        int size = 0,value = 0;
        bool bRet = false;

        Console.WriteLine("Enter the size of the array : ");
        size = int.Parse(Console.ReadLine());

        ArrayX obj = new ArrayX(size);
        obj.Accept();
        obj.Display();

        Console.WriteLine("Enter the element to search : ");
        value = int.Parse(Console.ReadLine());

        bRet = obj.SearchNum(value);
        if(bRet == true)        
        {
            Console.WriteLine("Element is present in array : ");
        }
        else
        {
            Console.WriteLine("There is no such element..");
        }
    }
}

File: scala/program14.scala
import scala.io.StdIn.{readInt}

class ArrayX(var size : Int)
{
    var arr = new Array[Int](size)

    def Accept() : Unit = {
        println("Enter elements of the array :")
        for(i <- 0 until size)
        {
            arr(i) = readInt()
        }
    }

    def searchElement(search : Int): Boolean = {
        for(num <- arr) {
            if(num == search) return true
        }
        false
    }
}

object Demo 
{
    def main(args : Array[String])
    {
        println("Enter number of elements : ")
        var size = readInt()

        var obj = new ArrayX(size)
        obj.Accept()

        println("Enter element search in array : ")
        var search = readInt()

        var res = obj.searchElement(search)
        if(res) {
            println("Element is present ")
        } else {
            println("Element is not present.")
        }
    }
}